ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -722,7 +752,23 @@ public class BottomSheetController: UIViewController, Shadowable, TokenizedContr
     // The output is only meaningful once view.bounds is non-zero i.e. a layout pass has occured.
     private func sheetFrame(offset: CGFloat) -> CGRect {
         let availableWidth: CGFloat = view.bounds.width
-        let sheetWidth = shouldAlwaysFillWidth ? availableWidth : min(Constants.maxSheetWidth, availableWidth)
+        let maxWidth = min(Constants.maxSheetWidth, availableWidth)
+
+        // Width will the fill the screen if should always fill width
+        // Otherwise we will try and set the size to the preferred width as long as its between the max and min width
+        // If its not between those we will make the maximum width size
+        let sheetWidth: CGFloat = {
+            let determinedWidth: CGFloat
+            if shouldAlwaysFillWidth {
+                determinedWidth = availableWidth
+            } else if Constants.minSheetWidth...maxWidth ~= preferredWidth {
+                determinedWidth = preferredWidth
+            } else {
+                determinedWidth = maxWidth
+            }
+            return determinedWidth
+        }()
```

Comment:
Should we just split this out to a function? I thought in some cases we've seen binary size impact to using closures and nested functions as opposed to more standard functions, especially when exposed to objC. But also good to just simplify and be consistent with the codebase

Or doesn't need to be a function, can just keep lines 761-769 and rename `determinedWidth` to `sheetWidth`